opm/autodiff/RedistributeDataHandles.hpp

```diff
@@ -324,7 +324,7 @@ void distributeGridAndData( Dune::CpGrid& grid,
     global_grid.switchToGlobalView();
 
     // distribute the grid and switch to the distributed view
-    grid.loadBalance(eclipseState);
+    grid.loadBalance(eclipseState, 1);
```

Comment:
Does this have anything to do with wells being null or is it needed for some other reason?  The reason I ask is that the commit message does not appear to cover this particular one-line change.